nodedb.go

```diff
@@ -234,6 +250,17 @@ func (ndb *nodeDB) isFastStorageEnabled() bool {
 	return ndb.getStorageVersion() >= fastStorageVersionValue
 }
 
+func (ndb *nodeDB) shouldForceFastStorageUpdate() bool {
+	versions := strings.Split(ndb.storageVersion, fastStorageVersionDelimiter)
+
+	if len(versions) == 2 {
+		if versions[1] != strconv.Itoa(int(ndb.getLatestVersion())) {
+			return true
+		}
+	}
+	return false
```

Comment:
This should never happen under correct behavior right? Should we add a comment do a fmt.Println if this occurs?

Reply:
This should happen when we have already upgraded to fast storage and it matches the latest state to avoid redundantly upgrading fast nodes when they are already live

Reply:
This should also happen if we are on the default (non-fast) storage version. This method is independent from `hasUpgradedToFastStorage`.

To avoid confusion,

hasUpgradedToFastStorage - returns true if we have upgraded to fast storage, false if we are on default

shouldForceFastStorageUpdate - returns true if we have upgraded to fast storage but it does not match the live state, false otherwise